tests/phpunit/tests/compat/arrayFirst.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @group compat
+ *
+ * @covers ::array_first
+ */
+class Tests_Compat_arrayFirst extends WP_UnitTestCase {
+
+	/**
+	 * Test that array_first() is always available (either from PHP or WP).
+	 */
+	public function test_array_first_availability(): void {
+		$this->assertTrue( function_exists( 'array_first' ) );
+	}
+
+	/**
+	 * @dataProvider data_array_first
+	 *
+	 * @ticket 63853
```

Comment:
Placing the ticket annotation before the dataprovider annotation is more common in the test suite.

```suggestion
	 * @ticket 63853
	 *
	 * @dataProvider data_array_first
```